angular/test/apps/ng16/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist/out-tsc",
+    "forceConsistentCasingInFileNames": true,
+    "strict": true,
+    "noImplicitOverride": true,
+    "noPropertyAccessFromIndexSignature": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "sourceMap": false,
```

Comment:
I had to disable source maps for Cypress to work (would throw an exception when opening cypress). This clearly isn't an issue in CI for the other test apps. I should re-enable and see if it is just a local Cypress issue (perhaps and outdated dmg).